.github/workflows/beta.yml

```
@@ -3,7 +3,7 @@ name: beta
 on:
   push:
     branches:
-      - master
+      - master, dev
```

actionlint (1.7.7)

character ' ' is invalid for branch and tag names. ref name cannot contain spaces, ~, ^, :, [, ?, *. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)